### ISSUE TITLE
is this plugin abandoned? 

### ISSUE THREAD

**Issue by phiasco12** [cjpearson/cordova-plugin-keyboard#92]
is this plugin abandoned? 

nothing seems to work on iOS 12+

and there hasn't been any updates since iOS 11.

**Comment by cjpearson** [comment]
I wouldn't say abandoned, but it's admittedly not very actively maintained. It's mostly a case of when I have time or when I encounter issues myself. Ideally, this plugin would rarely need changes since its scope is so limited, but iOS updates have a habit of throwing a wrench into the works. You can sometimes avoid this by staying on a lower target SDK version, assuming you don't need any of the new features. 

I'll try to see if I can't get through a few higher-priority issues this week though.

**Comment by phiasco12** [comment]
@cjpearson That would be highly appreciated.

The only and the main issue with this plugin is the fact that the 'scrolling' function doesn't work anymore on iOS 12+. Basically, when the keyboard is shown in the app, the whole app is pushed up and is scrollable which defeats the object of using this plugin.

And its the only plugin that was reliable and was working perfectly fine. Relying on other plugins such as the ionic keyboard is not ideal for people like me who don't use ionic framework and also, they don't seem to offer some features that your plugin does.

I will personally buy you a few beers if you could get something out this week Connor. :)

**Comment by Aarbel** [comment]
Maybe could be great to add other people to admin / contribute to this lib ;)

**Comment by phiasco12** [comment]
@Aarbel do you have a working solution?

**Comment by Aarbel** [comment]
yes, invite other people from Adobe (Cordova) to participate to this repo

**Comment by phiasco12** [comment]
@Aarbel I'm not the developer. @cjpearson is the developer.

**Comment by phiasco12** [comment]
@cjpearson any joy with this?

**Comment by Mapiac** [comment]
> @cjpearson That would be highly appreciated.
> 
> The only and the main issue with this plugin is the fact that the 'scrolling' function doesn't work anymore on iOS 12+. Basically, when the keyboard is shown in the app, the whole app is pushed up and is scrollable which defeats the object of using this plugin.
> 
> And its the only plugin that was reliable and was working perfectly fine. Relying on other plugins such as the ionic keyboard is not ideal for people like me who don't use ionic framework and also, they don't seem to offer some features that your plugin does.
> 
> I will personally buy you a few beers if you could get something out this week Connor. :)

@phiasco12 try adjustPan?

**Comment by phiasco12** [comment]
> > @cjpearson That would be highly appreciated.
> > The only and the main issue with this plugin is the fact that the 'scrolling' function doesn't work anymore on iOS 12+. Basically, when the keyboard is shown in the app, the whole app is pushed up and is scrollable which defeats the object of using this plugin.
> > And its the only plugin that was reliable and was working perfectly fine. Relying on other plugins such as the ionic keyboard is not ideal for people like me who don't use ionic framework and also, they don't seem to offer some features that your plugin does.
> > I will personally buy you a few beers if you could get something out this week Connor. :)
> 
> @phiasco12 try adjustPan?

isn't adjustPan for Android only?

**Comment by phiasco12** [comment]
@cjpearson is it safe to assume that this issue is not going to get resolved or this plugin will never gonna be updated?

reason for asking is because if you are not going to do anything with this plugin anymore, we need to look for an alternative for our project.

Could you please respond either way so we all know where we stand with this plugin?

Thanks

**Comment by BrandOuellette** [comment]
Anyone know if there is fork or another plugin that does the same thing but works on the latest iOS?

**Comment by phiasco12** [comment]
> Anyone know if there is fork or another plugin that does the same thing but works on the latest iOS?

please let us all know if you find any.

**Comment by BrandOuellette** [comment]
> > Anyone know if there is fork or another plugin that does the same thing but works on the latest iOS?
> 
> please let us all know if you find any.

We've decided to use https://github.com/ionic-team/cordova-plugin-ionic-keyboard instead of this one for now as it seems more up-to-date.

**Comment by dancombs** [comment]
As some have stated...using the Ionic Keyboard plugin seems to be the longer term.

But if you need a temp solution...

I ended up doing something like this - where "myNavigator" is the top most element in the DOM tree.

===

            $('#input-search').blur(function(e) {
                var new_position = $("#myNavigator").offset();
                $('html, body').stop().animate({
                    scrollTop: new_position.top
                }, 500);
                e.preventDefault();
            });
